runtime/domain.c

```diff
@@ -355,11 +353,11 @@ void caml_init_domains(uintnat minor_heap_wsz) {
     caml_fatal_error("Minor_heap_max misconfigured for this platform");
```

Comment:
We have one place on L352 (just above this one), where we do:
 `caml_mem_round_up_pages(Minor_heap_max) != Minor_heap_max`
I believe we should be doing `Bsize_wsize` on both of the occurrences of `Minor_heap_max` here.